web-services-app/src/main/java/edu/unc/lib/boxc/web/services/rest/DownloadBulkController.java

```diff
@@ -0,0 +1,53 @@
+package edu.unc.lib.boxc.web.services.rest;
+
+import edu.unc.lib.boxc.auth.api.Permission;
+import edu.unc.lib.boxc.auth.api.models.AccessGroupSet;
+import edu.unc.lib.boxc.auth.api.services.AccessControlService;
+import edu.unc.lib.boxc.model.api.ids.PID;
+import edu.unc.lib.boxc.model.fcrepo.ids.PIDs;
+import edu.unc.lib.boxc.operations.impl.download.DownloadBulkRequest;
+import edu.unc.lib.boxc.operations.impl.download.DownloadBulkService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import static edu.unc.lib.boxc.auth.fcrepo.services.GroupsThreadStore.getAgentPrincipals;
+import static org.apache.jena.ext.com.google.common.net.HttpHeaders.CONTENT_DISPOSITION;
```

Comment:
Use constants from `org.springframework.http.HttpHeaders` instead, we don't really want a dependency on something in jena here